router/src/infer.rs

```diff
@@ -530,6 +546,18 @@ async fn batching_task(
     shared: Arc<Shared>,
     generation_health: Arc<AtomicBool>,
 ) {
+
+    let read_env_var = |key: &str, default: u32| -> u32 {
+        std::env::var(key).ok().map_or(default, |value| value.parse::<u32>().unwrap())
+    };
+    let prefill_bucket_size: u32 = read_env_var("PREFILL_BATCH_BUCKET_SIZE", 4);
+    let max_input_tokens: u32 = read_env_var("MAX_INPUT_TOKENS", 1024);
+    let use_adaptive_batching = !std::env::var("MAX_TOTAL_TOKENS_2_MAX_BATCH_SIZE_LIST".to_string()).unwrap_or("".to_string()).to_string().trim().is_empty();
```

Comment:
This can be simplified with:
let use_adaptive_batching = std::env::var("MAX_TOTAL_TOKENS_2_MAX_BATCH_SIZE_LIST")
    .unwrap_or_default()
    .is_empty();

Reply:
I think it should be like this:
let use_adaptive_batching = !std::env::var("MAX_TOTAL_TOKENS_2_MAX_BATCH_SIZE_LIST").unwrap_or_default().is_empty();
Thanks! updated